docs/using-baklava-in-react.stories.mdx

```diff
@@ -114,3 +114,55 @@ function MyComponent() {
 
 export default MyComponent;
 ```
+
+## Styling Components
+You can customize components with css variables or general theme variables.
+
+#### Inline CSS
+
+```jsx
+import { BlButton } from "@trendyol/baklava/dist/baklava-react";
+
+function MyComponent() {
+  const buttonStyle = {
+    "--bl-button-main-color": "rebeccapurple",
+    "--bl-button-main-hover-color": "purple",
+  }
```

Comment:
This is not intended behavior. We'll move those "private" variables somewhere else. In theory it's not possible to change a color of a button variant without setting related theme color. But since you can set theme variables for a specific scope in document, you can set it like that:

```suggestion
  const buttonStyle = {
    "--bl-color-primary": "rebeccapurple",
    "--bl-color-hover": "purple",
  }
```

Reply:
thanks for info. fixed